PhysicsTools/NanoAOD/python/nanoDQM_cff.py

```diff
@@ -118,6 +118,11 @@
 
 run2_miniAOD_80XLegacy.toModify(nanoDQM.vplots, IsoTrack = None)
 
+_Muon_plots_preRun3 = copy.deepcopy(nanoDQM.vplots.Muon.plots)
+_Muon_plots_preRun3.remove(Plot1D('mvaIDMuon_WP', 'mvaIDMuon_WP',  3, -0.5, 2.5, 'MVA-based ID selector WPs (1=MVAIDwpMedium,2=MVAIDwpTight)'))
```

Comment:
This line is causing problems. This would probably work, though I agree it's not very nice:
```
_Muon_plots_preRun3 = cms.VPSet()
for plot in nanoDQM.vplots.Muon.plots:
    if plot.name.value() != "mvaIDMuon_WP":
        _Muon_plots_preRun3.append(plot)
```